docs/agents/alternativeMessageTopicClassificationAgent.md

```diff
@@ -0,0 +1,29 @@
+This is an agent that performs the message classification using [logic rules](../subsystems/scl-machine.md).
+
+**Action class:**
+
+`action_alternative_message_topic_classification`
+
+**Parameters:**
+
+1. `message node` - an element of `concept_message`;
+
+### Examples
+
+Examples of an input structure:
+
+<img src="../images/alternativeMessageTopicClassificationAgentInput.png"></img>
+
+Example of a logic rule to classify message:
+
+<img src="../images/lr_greeting_message.png"></img>
+
+### Agent implementation language
+C++
+
+### Result
```

Comment:
There is no output structure in this agent

Reply:
Hmm... It's true, but it sounds strange. It would still have to somehow save the message topic in the knowledge base. Maybe it doesn't have a direct output, but a 5-element structure is still generated in the process? I would indicate this in the documentation if that's the case.